engines/glk/magnetic/magnetic.h

```diff
@@ -1385,14 +1386,20 @@ class Magnetic : public GlkAPI {
 	/**
 	 * The Magnetic engine currently doesn't support loading savegames from the GMM
 	 */
-	bool canLoadGameStateCurrently() override {
+	bool canLoadGameStateCurrently(Common::U32String *msg = nullptr) override {
+		if (msg)
+			*msg = _("This game does not support loading from the menu. Use in-game interface");
```

Comment:
We should use the Engine::kSupportsLoadingDuringRuntime feature flag instead of this - unless there's a specific reason not to do that in this case?

Reply:
You can't. Magnetic does not exist as a separate engine.

Reply:
Indeed, thanks for the explanation!